src/main/kotlin/com/kotlin/boardproject/domain/user/repository/UserRepository.kt

```diff
@@ -1,9 +1,9 @@
-package com.kotlin.boardproject.repository
+package com.kotlin.boardproject.domain.user.repository
 
-import com.kotlin.boardproject.model.User
+import com.kotlin.boardproject.domain.user.domain.User
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end